UX-Guide-Metadata/draft/README.md

```diff
@@ -1,6 +1,6 @@
-# User Experience Guide for Displaying Accessibility Metadata
+# User Experience Guide for Displaying Accessibility Metadata 2.0
  
 * [Current Working Draft - Principles](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/)
```

Comment:
I'd suggest making sure you're not using "working draft" to describe these documents, like here, as that's a term for working group documents. You should call these editor's drafts.

When published they're either draft CG notes, if you're still planning to work on them, or CG notes if they're the final docs.